### DISCUSSION TITLE
Ukrainian Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#410] [Language Leaders]
@skynetua - are you interested in being a language leader for Ukrainian?

**Comment by skynetua** [comment]
Yes, why are you asking?

**Reply by synesthesiam** [reply]
I must have got you mixed up on my list. Please disregard :+1:

**Comment by synesthesiam** [comment]
@skynetua when you get a chance, could you please look into translating the `HassGetState` sentences/responses?

**Reply by skynetua** [reply]
I'll try to finish it today
